src/metabase/api/dashboard.clj

```diff
@@ -702,7 +701,19 @@
           changes-stats              (atom nil)
           ;; tabs are always sent in production as well when dashcards are updated, but there are lots of
           ;; tests that exclude it. so this only checks for dashcards
-          update-dashcards-and-tabs? (contains? dash-updates :dashcards)]
+          update-dashcards-and-tabs? (contains? dash-updates :dashcards)
```

Comment:
nit - not your fault, but this `let` is using a mixture of aligned and unaligned forms...can we stick to one or the other?

Reply:
I also feel like this function is getting pretty unwieldy. Maybe the `dash-updates` logic can be pulled out, or something.